src/metabase/search/impl.clj

```diff
@@ -664,7 +664,8 @@
    [:table-db-id                         {:optional true} [:maybe ms/PositiveInt]]
    [:search-native-query                 {:optional true} [:maybe true?]]
    [:model-ancestors?                    {:optional true} [:maybe boolean?]]
-   [:verified                            {:optional true} [:maybe true?]]])
+   [:verified                            {:optional true} [:maybe true?]]
+   [:ids                                 {:optional true} [:maybe [:set ms/PositiveInt]]]])
```

Comment:
Why not enforce that the set (if there is one) is non-empty here, like you do in `SearchContext`?

Reply:
IDK just did what every else similar line is doing. TBH this duplication bothers me a bit, but not enough right now. :)